src/click_handler.coffee

```diff
@@ -18,12 +18,18 @@ define [
       if !$target.is(@clickBindSelector)
         $target = $target.parent()
 
+      targetType = ($target, @finder) ->
+        if @finder.item($target) == 'phone_number_link'
+          'lead_submission'
+        else
+          'click'
```

Comment:
This looks good to me. I'm not sure if we should hard-code the item, or if we should pass it into the project as a configuration option. @c0 and @aribouius: do either of you have any strong feelings on this? We could hard-code it now, and refactor it to a configuration option later if the need arises.